d3rlpy/base.py

```diff
@@ -469,12 +550,15 @@ def fit(
                 self._evaluate(eval_episodes, scorers, logger)
 
             # save metrics
-            logger.commit(epoch, total_step)
+            metrics = logger.commit(epoch, total_step)
 
             # save model parameters
             if epoch % save_interval == 0:
                 logger.save_model(epoch, self)
 
+            metrics["epoch"] = epoch
+            yield metrics
```

Comment:
I'd like to return this tuple.
```
yield epoch, metrics
```

Reply:
Yes perfect, that was my first approach then a switched to including epoch inside since the result dic can then be converted directly to a pandas DataFrame, however with this it can be done as well setting epoch as index.

Reply:
I have done my best with the test code, but sorry this is not my strong skill so let see if what I included is enough.